lib/browser/api/web-contents.ts

```diff
@@ -343,8 +343,8 @@ WebContents.prototype.loadFile = function (filePath, options = {}) {
 
 type LoadError = { errorCode: number, errorDescription: string, url: string };
 
-WebContents.prototype.loadURL = function (url, options) {
-  const p = new Promise<void>((resolve, reject) => {
+WebContents.prototype._awaitNextLoad = function (navigationUrl: string) {
```

Comment:
To avoid _abuse_ can we make this truly private. i.e.

`function awaitNextLoad(...`

```
return awaitNextLoad.call(this)
```